packages/code-du-travail-frontend/src/content/index.js

```diff
@@ -0,0 +1,18 @@
+import * as Idcc from "./Idcc";
+import * as Faq from "./Faq";
+import * as CodeDuTravail from "./CodeDuTravail";
+import * as Kali from "./Kali";
+import * as FicheServicePublic from "./FicheServicePublic";
+import * as FicheMinistere from "./FicheMinistere";
+import * as ModeleDeCourrier from "./ModeleDeCourrier";
+
+// source -> component mapping
+export default {
+  faq: Faq,
```

Comment:
why not call it `question` ? so we can get rid of the urlByTosourceType ?

Reply:
indeed!